inst/apps/connect/app.R

```diff
@@ -14,8 +14,11 @@ BRAND_COLORS <- list(
   GRAY = "#404041"
 )
 
-# Load app constants
-source("../constants.R")
```

Comment:
When publishing to Connect from github, all files must be in the same parent directory, so we can't reference files at `..`. The `constants.R` file is not providing a ton of value, so I just moved the contents here.